meta-balena-common/recipes-containers/balena/balena_git.bb

```diff
@@ -11,10 +11,10 @@ LIC_FILES_CHKSUM = "file://src/import/LICENSE;md5=9740d093a080530b5c5c6573df9af4
 
 inherit systemd go pkgconfig useradd
 
-BALENA_VERSION = "18.09.8-dev"
-BALENA_BRANCH= "master"
+BALENA_VERSION = "18.9.10-dev"
+BALENA_BRANCH= "a2o-migrate"
```

Comment:
I'm afraid, I'd like to avoid switching the branch. Any reason why a2o-migrate isn't in master?

Reply:
this is not permanent. this pr is still a draft and so is https://github.com/balena-os/balena-engine/pull/168 because I still haven't actually tested this with the hoooks v2 stuff